app/packages/forms/imports/format_helpers.coffee

```diff
@@ -0,0 +1,49 @@
+formatDate = (dateString) ->
+  moment(dateString).format('MM/DD/YYYY hh:mm:ss')
+
+formatQuestionType = (type) ->
+  switch type
+    when 'number' then 'Number'
+    when 'scale' then 'Scale'
+    when 'multipleChoice' then 'Multiple Choice'
+    when 'checkboxes' then 'Checkbox'
+    when 'shortAnswer' then 'Short Answer'
+    when 'longAnswer' then 'Long Answer'
+    when 'date' then 'Date'
+    when 'datetime' then 'DateTime'
+    else
+      type
+
+formatAnswer = (answer, type) ->
+  switch type
+    when 'shortAnswer' then escapeString(answer)
+    when 'longAnswer' then escapeString(answer)
+    when 'date' then formatDate(answer)
+    when 'datetime' then formatDate(answer)
+    when 'multipleChoice' then escapeString(answer)
+    when 'checkboxes' then escapeString(answer.join ',')
+    else
+      answer
+
+  if type in [ 'shortAnswer', 'longAnswer' ]
```

Comment:
Good call I'll take care of that and move `formatQuestionType` to the helpers package so it can be used to format the type string in the `basic_results_info` template.